modules/targetanalysis/CMakeLists.txt

```diff
@@ -1,8 +1,12 @@
-find_path(Jsoncpp_INCLUDE_DIR
-  NAMES json/features.h
-  PATH_SUFFIXES jsoncpp
-  PATHS ${Jsoncpp_PKGCONF_INCLUDE_DIRS}
-)
+# find_path(Jsoncpp_INCLUDE_DIR
+#  NAMES json/features.h
+#  PATH_SUFFIXES jsoncpp
+#  PATHS ${Jsoncpp_PKGCONF_INCLUDE_DIRS}
+# )
```

Comment:
Since this is a VCS and stores code history, commenting out code rather than deleting it doesn't really serve a useful purpose.